cmd/epp/main.go

```diff
@@ -67,7 +67,12 @@ var (
 	targetEndpointKey = flag.String(
 		"targetEndpointKey",
 		runserver.DefaultTargetEndpointKey,
-		"Header key used by Envoy to route to the appropriate pod. This must match Envoy configuration.")
+		"Header and response metadata key used by Envoy to route to the appropriate pod. This must match Envoy configuration.")
+	targetEndpointOuterMetadataKey = flag.String(
+		"TargetEndpointOuterMetadataKey",
```

Comment:
It represents the receiving namespace of envoy, i think the name is a little confusing